templates/textfield.php

```diff
@@ -1,8 +1,16 @@
+<?php
+/**
+ * Template file for a textfield
+ *
+ * @package Templates
```

Comment:
Similarly, `@package Templates` would cause us to have different `@package`s within the plugin template parts. I used `@package Fieldmanager` in the Datepicker and Checkboxes templates, but I think that was a mistake. Something like `Fieldmanager\Template` might better fit [the existing hierarchy](http://api.fieldmanager.org/).